### PR TITLE
Update catch to 2.13.0 and adjust test code where necessary

### PR DESCRIPTION
**Pull request by mak-ableton**

### CODE REVIEW

File: src/test/tst_Convert.cpp [hunk 1]
```diff
@@ -77,7 +77,7 @@ TEST_CASE("Convert to font", "[convert]")
   auto f = convertProperty<QFont>(
     PropertyValue(std::string("italic mixedcase light nohinting 12pt Times New Roman")));
 
-  REQUIRE(f);
+  REQUIRE(f.has_value());
```
Comment (gck-ableton) [inline src/test/tst_Convert.cpp:80]: 🐦  Typo: "herre" in commit message

**Review by gck-ableton — COMMENTED** [review]
Looks good!

**Review by ala-ableton — APPROVED** [review]
LGTM 👍

**Review by gck-ableton — COMMENTED** [review]
👍

**Pull request merged**